projects/rocprofiler-systems/docker/Dockerfile.ubuntu.ci

```
@@ -17,9 +17,9 @@ ARG BOOST_DOWNLOAD_VERSION="1.79.0"
 ARG NJOBS="8"
 
 ENV PATH="/usr/local/bin:${PATH}"
-ENV LIBRARY_PATH="/usr/local/lib:/usr/local/lib64:${LIBRARY_PATH}"
-ENV LD_LIBRARY_PATH="/usr/local/lib:/usr/local/lib64:${LD_LIBRARY_PATH}"
-ENV CMAKE_PREFIX_PATH="/usr/local:${CMAKE_PREFIX_PATH}"
+ENV LIBRARY_PATH="/usr/local/lib:/usr/local/lib64"
+ENV LD_LIBRARY_PATH="/usr/local/lib:/usr/local/lib64"
+ENV CMAKE_PREFIX_PATH="/usr/local"
```

```suggestion
ENV LIBRARY_PATH="/usr/local/lib:/usr/local/lib64:${LIBRARY_PATH:-}"
ENV LD_LIBRARY_PATH="/usr/local/lib:/usr/local/lib64:${LD_LIBRARY_PATH:-}"
ENV CMAKE_PREFIX_PATH="/usr/local:${CMAKE_PREFIX_PATH:-}"
```
Would something like this work better? This way, if the base image changes to include the corresponding *_PATH variable, we won't overwrite it.

---

I did try that, but I got the same Docker warnings both when building locally and in the PR itself.

For now, I'll just leave them removed to avoid the warnings from appearing. It seems that the warnings don't understand the use-case there, even though it's logically sound and would avoid undeclared variables.